docs/build-haiku.md

```diff
@@ -0,0 +1,44 @@
+Haiku builds can be created using Clang or GCC using Meson buildsystem.
+
+## Build on Haiku
+
+Install dependencies:
+
+``` shell
+pkgman install -y meson ccache libpng libsdl2_devel sdl2_net_devel \
+                  libogg_devel opus_devel opusfile_devel gcc_syslibs_devel \
+                  opus_tools fluidsynth2_devel llvm_clang
+```
+
+Clone and enter the repository's directory:
+
+``` shell
+git clone https://github.com/dosbox-staging/dosbox-staging.git
+cd dosbox-staging
+```
+
+Build with OpenGL disabled (Haiku supports OpenGL only via software fallback):
+
+``` shell
+meson setup -Duse_opengl=false build
+ninja -C build
+```
+
+Detailed instructions and build options are documented in [BUILD.md](/BUILD.md).
+
+## Run
+
+Edit your configuration file by running: `dosbox -editconf` and make the
+following suggested changes (leave all other settings as-is):
+
+``` ini
+[sdl]
+output = texture
+texture_renderer = software
+
+[cpu]
+core = normal
+```
+
```

Comment:
"With OpelGL's shaders not available to scale the window, you might be interested in using a 2x or 3x scalar, as follows:

``` init
[render]
scaler = normal2x   # or normal3x
```

Reply:
Let's not make it a DOSBox tutorial… :)